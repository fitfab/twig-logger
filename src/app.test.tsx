import React from "react";
import { shallow } from "enzyme";

import App from "./app";

test("App Component renders", done => {
  const wrapper = shallow(<App title="Testing" />);
  expect(wrapper).toMatchSnapshot();
  expect;
  done();
});
