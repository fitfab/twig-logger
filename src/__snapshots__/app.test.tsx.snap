// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component renders 1`] = `
<styled.div>
  <styled.header>
    <Component
      color="#00BCD4"
      margin="0 10px 0 0"
    />
    <h1>
      Page Fingerprint
    </h1>
  </styled.header>
</styled.div>
`;
